<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="mapTab">
      <attribute name="title">
       <string>Map info</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="gameTab">
      <attribute name="title">
       <string>Game control</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
     <widget class="QWidget" name="logTab">
      <attribute name="title">
       <string>Actions log</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
